app/models/league.rb

```diff
@@ -1,5 +1,4 @@
 class League < Event
   enum game_modes: [:round_robin, :two_halfs, :swiss, :danish]
-
-  validates :name, :discipline, :game_mode, :max_teams, presence: true
+  # validates :name, :discipline, :game_mode, presence: true
```

Comment:
why is this commented and max_teams removed? Pretty sure they should be required

Reply:
This caused an issue earlier with double validation errors when submitting the create event form which resolved somehow. Is there any reason to include the same validation here if we already have it in the Event class?